SkietbaanBE/SkietbaanBE/appsettings.Development.json

```diff
@@ -6,5 +6,8 @@
       "System": "Information",
       "Microsoft": "Information"
     }
+  },
+  "ConnectionStrings": {
+    "SkietbaanDatabase": "Server=dev.retrotest.co.za;Database=skietbaan;User Id=group3;Password=Y%6KG-bmr96Pe5rC;"
```

Comment:
You should not commit such sensitive data into your repo.

Reply:
add this as  a server environmental variable please. this is not safe.